### PR TITLE
added context processor and tests

### PR DESCRIPTION
**Pull request by aethemba**

### CODE REVIEW

File: tenant_extras/context_processors.py [hunk 1]
```diff
@@ -1,23 +1,60 @@
...
         properties = get_tenant_properties()
-
         return {
             'DONATIONS_ENABLED': getattr(properties, 'DONATIONS_ENABLED'),
             'RECURRING_DONATIONS_ENABLED': getattr(properties, 'RECURRING_DONATIONS_ENABLED'),
             'TENANT': connection,
             'TENANT_LANGUAGE': '{0}{1}'.format(current_tenant.client_name, request.LANGUAGE_CODE),
-            'LANGUAGES': json.dumps([{'code': lang[0], 'name': lang[1]} for lang in getattr(properties, 'LANGUAGES')])
-        }
+            'LANGUAGES': json.dumps([{'code': lang[0], 'name': lang[1]} for lang in getattr(properties, 'LANGUAGES')]),
```
Comment (iivvoo) [inline tenant_extras/context_processors.py]: unnecessary change: comma too much

**Comment by aethemba** [comment]
Good point, comma is removed
